### ISSUE TITLE
"Login unsuccesfull: Unable to login, please check your credentials" after update to HA 2022.05 & Indego 3.8.1

### ISSUE THREAD

**Issue by 35aportal** [jm-73/Indego#158]
Hello,
I've enjoyed seamless integration until I updated both HA and the Indego module this morning. Now I see...

    Logger: custom_components.indego
    Source: custom_components/indego/__init__.py:233
    Integration: Bosch Indego Mower (documentation)
    First occurred: 09:30:47 (1 occurrences)
    Last logged: 09:30:47

   Login unsuccesfull: Unable to login, please check your credentials

Nothing else changed between a working integration before the update and I can confirm that I have access to my Bosch account using the same details in the Android app.

I'd be very grateful of your advice on how to fix this.

**Comment by robbse81** [comment]
Same here.

**Comment by user45876** [comment]
And the same here, excellent description by OP.

**Comment by mhosse** [comment]
I had the same issue. But after another HA restart (to activate debug logging) it now works again.

**Comment by robbse81** [comment]
> I had the same issue. But after another HA restart (to activate debug logging) it now works again.

I can confirm that. Thanks.

**Comment by 35aportal** [comment]
Yup , another restart worked for me also. Many thanks for sharing your discovery.
